vlib/pg/README.md

```diff
@@ -1,13 +1,12 @@
 ## Description:
 
-`pg` is a wrapper for the PostgreSQL client library. It provides access
-to a PostgreSQL DB server.
+`pg` is a wrapper for the PostgreSQL client library. It provides access to a PostgreSQL DB server.
```

Comment:
```suggestion
`pg` is a wrapper for the PostgreSQL client library. It provides access to a PostgreSQL database server.
```
In formal docs, it is better to use the full words before introducing abbreviations.